examples/example_offresonance.py

```diff
@@ -0,0 +1,130 @@
+"""
+======================
+Off-resonance Corrected NUFFT Operator
+======================
+
+Example of Off-resonance Corrected NUFFT trajectory operator.
+
+This examples show how to use the Off-resonance Corrected NUFFT operator to acquire 
+and reconstruct data in presence of field inhomogeneities.
+Here a spiral trajectory is used as a demonstration.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from mrinufft import display_2D_trajectory
+
+plt.rcParams["image.cmap"] = "gray"
+
+# %%
+# Data Generation
+# ===============
+# As example, 2D shepp-logan will be used.
+# installable using ``pip install phantominator``
+
+from phantominator import ct_shepp_logan
+
+mri_data = ct_shepp_logan(256)
+plt.imshow(mri_data), plt.axis("off")
+
+# %%
+# Field Generation
+# ===============
+# Here, we generate a radial B0 field with the same shape of
+# the input Shepp-Logan phantom
+
+
+def make_b0map(obj, b0range=(-300, 300)):
+    """Make radial B0 field.
+
+    Parameters
+    ----------
+    obj : np.ndarray
+        Input object of shape (ny, nx).
+    b0range : tuple, optional
+        B0 field range in [Hz]. The default is (-300, 300).
+
+    Returns
+    -------
+    b0map : np.ndarray
+        Field inhomogeneities map of shape (ny, nx)
+    """
+    # calculate grid
+    ny, nx = obj.shape
+    yy, xx = np.mgrid[:ny, :nx]
+    yy, xx = yy - ny // 2, xx - nx // 2
+    yy, xx = yy / ny, xx / nx
+
+    # radius
+    rr = (xx**2 + yy**2) ** 0.5
+
+    # mask
+    mask = (obj != 0).astype(np.float32)
+    b0map = mask * rr
+
+    # rescale
+    b0map = (b0range[1] - b0range[0]) * b0map / b0map.max() + b0range[0]  # Hz
+
+    return b0map * mask
```

Comment:
This can be a function / fixture can can be used in tests too!

Reply:
I am not really sure how to avoid copying the code here and in tests without incorporating these in the mrinufft._utils. Do you have any suggestion?

Reply:
Maybe in a `mrinufft/extras/data.py` file ?

Reply:
Done, thanks!